DiscordChatExporter.Core.Models/Guild.cs

```diff
@@ -10,13 +14,19 @@ public partial class Guild : IHasId
 
         public string? IconHash { get; }
 
+        public List<Role> Roles { get; }
```

Comment:
Change this to `IReadOnlyList<Role>` for portability and consistency (will also have to update constructor).